examples/funloc/analysis_fun.py

```diff
@@ -118,10 +115,17 @@
     [0, 1],  # Only ensure the standard event counts match
 ]
 
-params.report_params['coil_snr'] = False
+params.report_params.update(  # add a couple of nice diagnostic plots
+    whitening=dict(analysis='All', name='All',
+                   cov='%s-55-sss-cov.fif'),
+    source=dict(analysis='All', name='All',
+                inv='%s-55-sss-meg-eeg-free-inv.fif', times=[0.1, 0.2],
+                views='lat', size=(800, 400)),
+    psd=False,  # often slow
```

Comment:
Filter setting should be grabbed from `params`. And I don't have low pass in my inverse filenames, is that a new addition?

Reply:
What do you mean about the filter setting? This is just whether or not to plot the PSD in the report.

According to `git blame` the frequency has been there at least 3 years (but probably longer):

https://github.com/LABSN/mnefun/blame/master/mnefun/_paths.py#L167

Reply:
Sorry, disregard this one. I fukced up.